nemo/collections/nlp/models/language_modeling/megatron/bert_model.py

```diff
@@ -197,14 +212,17 @@ def __init__(
             use_cpu_initialization=use_cpu_initialization,
             precision=precision,
             fp32_residual_connection=fp32_residual_connection,
-            activations_checkpoint_method=activations_checkpoint_method,
-            activations_checkpoint_num_layers=activations_checkpoint_num_layers,
+            activations_checkpoint_granularity=None,
+            activations_checkpoint_method=None,
+            activations_checkpoint_num_layers=1,
+            activations_checkpoint_layers_per_pipeline=None,
```

Comment:
Shouldn't num_micro_batches_with_partial_activation_checkpoints also be set somewhere here? Not sure